iFixit/react-commerce
# feat: disable zoom for products with low res images

closes #1097 

Disable image zoom when the image size is below a certain threshold.

### Edit
As discussed in weekly call we agreed on enabling the zoom if the image natural size is larger than the containing element.

If only one image in `product.images` is below threshold the whole zoom functionality is disabled.
This is because it emerged that there will be almost no product with both low and high res images.

### QA

1. Visit Vercel preview ([here](https://react-commerce-git-disable-zoom-for-products-with-dd7f23-ifixit.vercel.app/products/iphone-11-screen) I inserted a low res image in the part only variant)
2. Verify that products with even just one image with `intrinsic size width * intrinsic size height < container width * container height` have zoom disabled
3. Verify that the enabling state might be mutated by resizing the containing element (this take into account possible rotations of the screen on mid-sized devices)
